src/jquery.daterangepicker.js

```diff
@@ -1803,7 +1803,7 @@
         }
 
         function countDays(start, end) {
-            return Math.abs(daysFrom1970(start) - daysFrom1970(end)) + 1;
+            return moment(start).diff(moment(end), 'd') + 1;

```

Comment:
Add `Math.abs`, because returning value can be negative;

Reply:
Also you can remove functions `daysFrom1970` and `toLocalTimestamp`, they not required anymore.